CHANGELOG.rst

```diff
@@ -10,6 +10,10 @@ Added
   to ensure update to the models require schema to be regenerated. (new feature)
 * Improved st2sensor service logging message when a sensor will not be loaded when assigned to a
   different partition (@punkrokk)
+* Add support for a configurable connect timeout for SSH connections as requested in #4715
+  by adding the new configuration parameter ``ssh_connect_timeout`` to the ``ssh_runner``
+  group in st2.conf.
+  This option was requested by Harry Lee (@tclh123) and contributed by Marcel Weinberg (@winem).
```

Comment:
Best way to verify your changelog format is by comparing it with the records from previous releases.

```suggestion
* Add support for a configurable connect timeout for SSH connections as requested in #4715
  by adding the new configuration parameter ``ssh_connect_timeout`` to the ``ssh_runner``
  group in st2.conf. (new feature) #4914
  
  This option was requested by Harry Lee (@tclh123) and contributed by Marcel Weinberg (@winem).
```

It looks like we got several misleading changelog records in this release which show consistency issues cc @punkrokk.

Reply:
Done. Sorry, will pay more attention to the details next time.

Reply:
looks like there is still a diff.

Reply:
Done. Forgot to push again.

Reply:
Thanks! :+1: